src/content/guidelines/accessibility/color.md

```diff
@@ -27,11 +26,11 @@ Color blindness involves difficulty in perceiving or distinguishing between colo
 | Monochromacy | All colors |  
 
 ### Designing for color blindness
-When designing color palettes, it's always good to run them through a color blind simulator. If you're working in Sketch, we recommend the [Stark](http://www.getstark.co/) plugin.
+When designing with color, it's always good to run your pages or comps through a color-blind simulator to ensure that all functions are still clear and usable by the color-blind user. If you're working in Sketch, we recommend the [Stark](http://www.getstark.co/) plugin.  
```

Comment:
comps feels casual in this context, would it help to consolidate this phrase to just `good to run your work`?